pkg/operator/deploy/deploy_test.go

```diff
@@ -0,0 +1,126 @@
+package deploy
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"testing"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/util/cmp"
+)
+
+func TestIngressProfiles(t *testing.T) {
```

Comment:
Maybe this naming makes more sense?
```suggestion
func TestGetIngressIP(t *testing.T) {
```